response_writer.go

```diff
@@ -96,39 +89,8 @@ func (rw *responseWriter) Before(before func(ResponseWriter)) {
 	rw.beforeFuncs = append(rw.beforeFuncs, before)
 }
 
-func (rw *responseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
```

Comment:
Same here, I think this interface break should prompt us to bump the major version.